src/Payload.cpp

```diff
@@ -43,6 +43,13 @@ std::string Payload::moveDataToString() {
   return data->moveToFbString().toStdString();
 }
 
+std::string Payload::cloneDataToString() const {
```

Comment:
implementing the `<<` overload is a bit more idiomatic (and yields nicer code). Like here: https://github.com/ReactiveSocket/reactivesocket-cpp/blob/f1b862a23bc951bbe5869d03321eafa4f583a2a1/src/Common.h#L32